src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java

```diff
@@ -109,6 +109,11 @@ public void setUp() throws Exception {
         doReturn(gt).when(trigger).getGitHub();
         
         given(gt.getRepository(anyString())).willReturn(ghRepository);
+
+        // Set rate limit before starting trigger
+        GHRateLimit initRateLimit = new GHRateLimit();
```

Comment:
It fixes a NullPointerException being thrown from the GhprbTrigger.start method.  I at first tried moving the mock GHRateLimit up to before the start was called but it then messes with the number of times the rate limit was called when `verifyGetGithub(callsCount, repoTimes)` is called during the tests themselves.

So instead I elected to just give it a temporary one so it could call the start method without failure and then replace it with the mock one that the tests use to verify the number of times the various methods are called.

`java.lang.NullPointerException
	at org.jenkinsci.plugins.ghprb.GhprbRepository.initGhRepository(GhprbRepository.java:99)
	at org.jenkinsci.plugins.ghprb.GhprbRepository.init(GhprbRepository.java:71)
	at org.jenkinsci.plugins.ghprb.GhprbTrigger.initState(GhprbTrigger.java:221)
	at org.jenkinsci.plugins.ghprb.GhprbTrigger.start(GhprbTrigger.java:242)`